remoteappmanager/static/js/home/controller.js

```diff
@@ -51,5 +51,7 @@ require([
         ga_observer.trigger_application_starting(application.app_data.image.name);
     });
 
+    app_list_view.$on('entry_clicked', function() { app_view.focus_iframe(); });
```

Comment:
I also tried `app_list_view.$on('click.native', myFunction)` according to the [documentation](https://vuejs.org/v2/guide/components.html#Using-v-on-with-Custom-Events) but it sounds like the `.native` modifier doesn't work for `$on`, but only for `v-on` in templates.